plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Can you document the fields here (comments)?

Reply:
Added reference to submitit. I thought about adding it line by line in hydra but it seems repetitive.